pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -268,6 +271,29 @@ public static boolean checkUserAccess(String realm, String userId, String role,
         return false;
     }
 
+    public static boolean checkUserAccessForPatch(String realm, String userId, String role, String userIdRequest, Patch patch) {
+        String userEmailOrIdInPatch = patch.getUser();
+        String userIdFromPatch;
+        if (userEmailOrIdInPatch.contains("@")) {
+            // patches come with both user id or email in them, this is to find the id in case it was an email
+            UserDto user = new UserDao().getUserByEmail(userEmailOrIdInPatch).orElseThrow();
+            userIdFromPatch = user.getId() + "";
```

Comment:
Do we typically use this idiom rather than a toString or similar method?

Reply:
Not sure what we use, different people have used different forms in the past.  I will change this to use toString and we can all stay consistent with that